src/main/java/AllMenuOptions.java

```diff
@@ -4,6 +4,7 @@
     DISPLAY_ALL_OPP("d", "Display all opportunities", startingMenu()),
     SEARCH_BY_ID("i", "Search by id", startingMenu()),
     UPDATE_OPP("u", "Update an opportunity", startingMenu()),
+    DELETE_OPP("d", "Delete an opportunity", startingMenu()),
```

Comment:
I'd have this as `DELETE_OPPORTNUITY` or `DELETE`. I've encountered code where people have written a short form of a word, and I've not known what they were talking about!

Reply:
on the flipside: I have an app that has opportunity as a 'object' record and `opp` has been used as shortening since it takes up such character count. But only as like vars within a function that has the full name.